gvsbuild/utils/builder.py

```diff
@@ -436,9 +436,18 @@ def __check_vs(self, opts):
                 "Looking for the Visual Studio version installed under %s ..."
                 % (opts.vs_install_path,)
             )
+
             for part in dir_parts:
+                if (
+                    part == "BuildTools"
+                    and opts.vs_install_path
+                    == r"C:\Program Files\Microsoft Visual Studio\2022"
```

Comment:
This seems a lot of special casing... should we at least programmatically get `Program Files` etc instead of hardcoding `C:` and the english name?

Also, may be a good idea to add a comment about this